doc/nrf/includes/ieee802154_eui64_conf.txt

```diff
@@ -0,0 +1,48 @@
+IEEE 802.15.4 EUI-64 configuration
+==================================
+
+An IEEE EUI-64 address consists of two parts:
+
+* Company ID - a 24-bit MA-L (MAC Address Block Large), formerly called OUI (Organizationally Unique Identifier)
+* Extension identifier - a 40-bit device unique identifier
+
+You can configure the EUI-64 for a device in the following ways:
+
+Use the default
+  By default, the company ID is set to Nordic Semiconductor's MA-L (``f4-ce-36``).
+  The extension identifier is set to the DEVICEID from the factory information configuration registers (FICR).
+
+Replace the company ID
+  You can enable :option:`CONFIG_IEEE802154_VENDOR_OUI_ENABLE` to replace Nordic Semiconductor's company ID with your own company ID.
+  Specify your company ID in :option:`CONFIG_IEEE802154_VENDOR_OUI`.
+
+  The extension identifier is set to the default, namely the DEVICEID from FICR.
+
+Replace the full EUI-64
+  You can provide the full EUI-64 value by programming certain user information configuration registers (UICR).
+  For nRF52 Series devices, the CUSTOMER registers block is used.
+  For nRF53 Series devices, the OTP registers block is used.
+
+  To use the EUI-64 value from the UICR, enable :option:`CONFIG_IEEE802154_UICR_EUI64_ENABLE` and set :option:`CONFIG_IEEE802154_UICR_EUI64_REG` to the base of the two registers that contain your EUI-64 value.
```

Comment:
```suggestion
  To use the EUI-64 value from the UICR, enable :option:`CONFIG_IEEE802154_UICR_EUI64_ENABLE` and set :option:`CONFIG_IEEE802154_UICR_EUI64_REG` to the base of the two consecutive registers that contain your EUI-64 value.
```
Do they have to be consecutive? Probably, since you're only specifying one value?